### ISSUE TITLE
Comparison between P2-Gamma and No P2-Gamma

### ISSUE THREAD

**Issue by Mut1nyJD** [lucidrains/denoising-diffusion-pytorch#70]
Hi Lucidrains,

Well a bit late but now I successfully conducted a test between using P2-Gamma while training and one without 

I made a little comparison video:

https://www.youtube.com/watch?v=3k68aI5QdVU

(Gamma weight is displayed in the video)

I trained my custom dataset for 30 epochs. Using cosine schedule, L2 norm and 500 timesteps
UNet structure has 128 channels base and using  multiplier 1,1,2,4. Image size of training image is 256^2

Well you have to go through the video frame by frame basically to see the what happens. I did not compute any FID metrics (yet). By eyeballing I would say P2-Gamma seems to converge to better results slightly early but it is really hard to say at least in this example

**Comment by askerlee** [comment]
What's the dataset? It's very hard to compare at an early stage. Some methods converge faster but to a less optimal state.

**Comment by martinriven** [comment]
I find it converge fast too, the loss looks great, but I can't use it to get a good result, wonder why?
